src/main/resources/wa-task-initiation-employment-et_englandwales.dmn

```diff
@@ -95,17 +95,20 @@ then additionalData.Data.respondentCollection[1].value.et3Vetting.et3ContractCla
 else null</text>
         </inputExpression>
       </input>
-      <input id="InputClause_16zvm0d" label="List A Hearing">
-        <inputExpression id="LiteralExpression_0p48kud" typeRef="boolean">
+      <input id="InputClause_16zvm0d" label="List A Hearing" biodi:width="150">
+        <inputExpression id="LiteralExpression_0p48kud" typeRef="boolean" expressionLanguage="feel">
           <text>if(additionalData.Data.etICCanProceed!=null
 and additionalData.Data.etICCanProceed
 and additionalData.Data.etICHearingAlreadyListed!=null)
 then if(additionalData.Data.etICHearingAlreadyListed)
 then additionalData.Data.etICHearingAlreadyListed
 else if(additionalData.Data.etICHearingNotListedList!=null
 and count(additionalData.Data.etICHearingNotListedList) &gt; 0
-and (list contains(additionalData.Data.etICHearingNotListedList, "List for preliminary hearing")
-or list contains(additionalData.Data.etICHearingNotListedList, "List for final hearing")))
+and (list contains(additionalData.Data.etICHearingNotListedList, "Seek comments on the video hearing")
+or list contains(additionalData.Data.etICHearingNotListedList, "List for preliminary hearing")
+or list contains(additionalData.Data.etICHearingNotListedList, "List for final hearing")
+or list contains(additionalData.Data.etICHearingNotListedList, "UDL hearing")
+or list contains(additionalData.Data.etICHearingNotListedList, "Do not list")))
```

Comment:
This should be a new column in the DMN as opposed to changing the existing one as we do not want to create the task 'List a Hearing' when selecting 'Do not list'

Reply:
corrected